guppylang/prelude/_internal/compiler/prelude.py

```diff
@@ -94,6 +94,24 @@ def build_unwrap_right(
     return conditional.to_node()
 
 
+def build_unwrap_left(
```

Comment:
seems worth commoning up with `build_unwrap_right`, only the case numbers differ

Reply:
We can do that once https://github.com/CQCL/hugr/issues/1596 is fixed

Reply:
leave a todo